drivers/sent/Kconfig

```diff
@@ -0,0 +1,23 @@
+# Copyright 2025 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig SENT
+	bool "Single Edge Nibble Transmission (SENT)"
+	help
+	  Enable SENT Driver Configuration
+
+if SENT
```

Comment:
How about `CONFIG_SAEJ2716`? Similarly to `IEEE802154`, `IEEE80211*`, etc.

Reply:
I replaced `SENT` to `SAEJ2716`